pkg/controllers/provisioningv2/cluster/controller.go

```diff
@@ -261,6 +274,8 @@ func mgmtClusterName() (string, error) {
 	return name.SafeConcatName("c", "m", rand[:8]), nil
 }
 
+// createNewCluster creates a clusters.management.cattle.io/v3 object based cluster of the
+// clusters.provisioning.cattle.io/v1 object and the spec of the clusters.management.cattle.io/v3 object passed in.
```

Comment:
Maybe it's just me, but reading through this comment I don't understand what the func is doing. Probably a typo / missing word?

Reply:
I read it again and realized I wrote junk, so I fixed it.